q2_feature_table/_merge.py

```diff
@@ -25,7 +25,8 @@ def _get_overlapping(tables, axis):
 
 def merge(tables: biom.Table,
           overlap_method: str = 'error_on_overlapping_sample') -> biom.Table:
-    if len(tables) == 1:
+    n_tables = len(tables)
```

Comment:
I figured no need to compute the length twice - and also I think it helps stitch it all together (I didn't even notice we were computing twice until I expanded the diff).

Reply:
That’s [fine](https://wiki.c2.com/?PrematureOptimization) :).

Thanks again.

Reply:
Hah! I don't know the first thing about optimization, let alone premature optimization.